src/main/java/higtools/modules/main/AutoCenter.java

```diff
@@ -0,0 +1,72 @@
+package higtools.modules.main;
+
+import higtools.HIGTools;
+import meteordevelopment.meteorclient.events.world.TickEvent;
+import meteordevelopment.meteorclient.settings.DoubleSetting;
+import meteordevelopment.meteorclient.settings.Setting;
+import meteordevelopment.meteorclient.settings.SettingGroup;
+import meteordevelopment.meteorclient.systems.modules.Module;
+import meteordevelopment.orbit.EventHandler;
+
+import static higtools.utils.HIGUtils.getHighway;
+
+public class AutoCenter extends Module {
+    // x+ x- z+ z- x+z+ x-z+ x+z- x-z-
+    private int highway = -1;
+    private final SettingGroup sgGeneral = settings.getDefaultGroup();
+
+    private final Setting<Double> comp = sgGeneral.add(new DoubleSetting.Builder()
+        .name("Compensation")
+        .defaultValue(1.0)
+        .description("How much to compensate for the player's position")
+        .range(-2.5, 2.5)
+        .sliderRange(-2.5,2.5)
+        .build()
+    );
+
+    private final Setting<Double> maxSpeed = sgGeneral.add(new DoubleSetting.Builder()
+        .name("MaxSpeed")
+        .defaultValue(0.1)
+        .description("How fast to try to get you on axis")
+        .range(0.01, 0.5)
+        .build()
+    );
+
+    public AutoCenter() {
+        super(HIGTools.MAIN, "AutoCenter", "Automatically centers the player so that NetherBorer mines on axis.");
+    }
+
+    @Override
+    public void onActivate() {
+        super.onActivate();
+        if (mc.player == null) return;
+        highway = getHighway();
+    }
+
+    @EventHandler
+    public void motions(TickEvent.Pre event) {
+        check();
+    }
+
+    @EventHandler
+    public void motions(TickEvent.Post event) {
+        check();
+    }
+
+    private void check() {
```

Comment:
In the `onActivate()` method, it gets the higwhay value from [this](https://github.com/Big-Iron-Cheems/HIGTools/blob/Java-rw/src/main/java/higtools/utils/HIGUtils.java#L136) method in `HIGUtils` class.

That method was pretty much the same in the original KT source, except for its location being another file.
Based on the X, Z coords it figures out on what axis the player is.

Reply:
Could it be possible to activate the right borer automatically depending on which axis the player is :trollface:

Reply:
I guess so, yes. It could be done in this method directly, but it would turn on the correct borer only when turning on AutoCenter. 

If that's not what you want, give me some more specs. It might even be better to implement that in a separate PR following this one instead.

Reply:
Perfect 👍